.swift-version

```diff
@@ -1 +1 @@
-DEVELOPMENT-SNAPSHOT-2016-03-24-a
+3.0-GM-CANDIDATE
```

Comment:
Use 3.0-GM-CANDIDATE to build by SPM
all of dependencies(Quick/Nimble/Result) uses this snapshot.